pinot-common/src/main/java/org/apache/pinot/common/datatable/DataTableImplV4.java

```diff
@@ -62,12 +62,23 @@ public class DataTableImplV4 extends BaseDataTable {
   protected byte[] _variableSizeDataBytes;
   protected ByteBuffer _variableSizeData;
   protected Map<String, String> _metadata;
-  private static final int VERSION = 1;
   protected int[] _columnOffsets;
   protected int _rowSizeInBytes;
 
   public DataTableImplV4() {
-    super();
+    _numRows = 0;
```

Comment:
not sure I can tell from just reviewing the PR, what was the bug?

Reply:
see my previous comment :-)